concourse/tasks/daisy-publish-images.yaml

```diff
@@ -12,13 +12,15 @@ inputs:
 
 params:
   GOOGLE_APPLICATION_CREDENTIALS: "credentials/credentials.json"
+  ROLLOUT_RATE: 60
 
 run:
   path: /gce_image_publish
   args:
   - -source_gcs_path=((source_gcs_path))
   - -source_version=((source_version))
   - -publish_version=((publish_version))
+  - -rollout_rate=$ROLLOUT_RATE
```

Comment:
this form of referencing environment variables is a shell feature. this container doesn't run a shell, it runs the binary `/gce_image_publish`. so this string will be read literally.

what is your testing mechanism for this change?

Reply:
Thank you for the feedback and pointers. I've updated the PR with new task in this case.